lib/puppet/pops/types/p_sem_ver_type.rb

```diff
@@ -97,6 +101,10 @@ def from_args(major, minor, patch, prerelease = nil, build = nil)
       def from_hash(hash)
         SemanticPuppet::Version.new(hash['major'], hash['minor'], hash['patch'], hash['prerelease'], hash['build'])
       end
+
+      def on_error(str)
+        "The string '#{str}' cannot be converted to a SemVer"
```

Comment:
None of the errors are handled with I18N support. (The earlier version did not have I18N either, but it should as these type conversion errors are user facing).  Will mark other occurrences with only "i18n"